src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
 }
 
 fn bits_per_word() -> usize {
     usize::max_value().count_ones() as usize
 }
 
-#[inline(never)]
-fn clone_slow(arg: &Box<Vec<usize>>) -> Box<Vec<usize>> {
-    arg.clone()
-}
-
 impl Clone for Bitset {
     #[inline]
     fn clone(&self) -> Bitset {
```

Comment:
I get the "inline the fast path", but not "force deinline the slow". Does it take longer to use `Bitset::clone` if the `Some` branch is longer (assuming it is never used)? That is, does the modification done here to inline `clone_slow` but leave the `match` expression alone make the code slower?